Demo/DemoTests/MoyaProviderSpec.swift

```diff
@@ -56,7 +56,29 @@ class MoyaProviderSpec: QuickSpec {
                         expect(endpoint1).to(equal(endpoint2))
                     }
                 })
-                
+
+                it("delays execution when appropriate") {
```

Comment:
Think I might move to `waitUntil` for all necessarily asynchronous tests, instead of `toEventually`.